README.md

```diff
@@ -6,6 +6,6 @@ This repository contains the source code for [The Encryption Compendium](https:/
 ## Development
 See [DEVELOPMENT.md](/DEVELOPMENT.md) for more information about how to develop the site. If you'd like to open an issue or make a pull request, please refer to [CONTRIBUTING.md](/CONTRIBUTING.md) for contribution guidelines.
 
-![Silicon Flatirons logo](/static/img/silicon_flatirons_logo.gif)
+![Silicon Flatirons logo](/static/img/SILC_logo.png)
 
-![CU Boulder TCP logo](/static/img/tcp_logo.png)
+![CU Boulder TCP logo](/static/img/Colorado_Law_rev_center_gold_law.png)
```

Comment:
```suggestion
![CU Boulder Law School logo](/static/img/Colorado_Law_rev_center_gold_law.png)
```

Reply:
LGTM -- i've updated the branch to reflect both the logo change and the text about the contributing organization.

Reply:
Ah, awesome -- thanks for making those changes!